### PR TITLE
1.4.2: Research Combat Zone (+ Various new loot and tweaks)

### PR DESCRIPTION
**Pull request by PKPenguin321**
- Version number bumped to 1.4.2
- Added a new combat zone with 4 new enemies relating to R&D
- Added 3 new items: Mech mounted drill, R&D jumpsuit, Bag of Holding
- Bugfix: Move speed can no longer underflow and is capped at 0.1 (this cannot actually happen yet but might in the future)
- R&D now benefits from XP boosts (currently R&D jumpsuit and chameleon jumpsuit)
- - XP from tasks AND item bounties is boosted
- XP bonuses for validhunting now change the value displayed on the bounty box before you claim it
- - Technical: Validhunting now applies XP bonuses sanely
- The setting to show underlines on actions you have not yet completed now underlines enemies you haven't defeated yet too
- Fluff: Most enemies now have a dialogue blurb viewable by mousing over their icon
image
- Technical: Companion slot items now have a variable that can be used to have 0 flee chance
- Technical: Equip requirements apply to XP bonuses
- Export Bonuses: Every 23 hours, you may instantly gain 30 minutes of banked Chrono Time by exporting your save. This works as an incentive/reminder to export often in case you wipe your save by resetting cookies.
image
- New Chrono Shop upgrade: Unlock Time Selling. Once you purchase this you can then trade 1 hour of banked chrono time for 25,000 money as often as you want for the rest of the loop.
- New chronoshop and cargo upgrades: Validhunting bounty rerolls. You can reroll your validhunting bounty once per upgrade per successful bounty.

### CODE REVIEW

File: src/data/enemies/dorms4.js [hunk 1]
```diff
@@ -135,6 +137,8 @@ export default {
 	},
 chemist: {
 		name: "Wandering Chemist",
+		desc: "Hey, you! Wanna try my unlabelled pills?",
```
Comment (Nabski89) [inline src/data/enemies/dorms4.js:140]: unlabeled

Comment (PKPenguin321) [reply src/data/enemies/dorms4.js:140]: American English moment

**Comment by PKPenguin321** [comment]
Last two commits add chrono shop stuff that I want to include for 1.4.2 but are technically out of scope of the PR (as it focuses mainly on combat-centric additions like the new combat zone). If you want me to break the PR up into multiple let me know, but if you don't mind we can keep it in one

File: src/data/enemies/dorms4.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export default {
 	catgirl: {
 		name: "Catboy",
+		desc: "I hope that's not a gun in your pocket... OwO",//banned! banned!!!!!
```
Comment (TBartl) [inline src/data/enemies/dorms4.js:4]: =_=

**Review by TBartl — APPROVED** [review]
looks good lets chat on discord on stuff soon

**Comment by PKPenguin321** [comment]
Gonna test out my merge/AWS perms

**Pull request merged**